portfolio/src/main/webapp/script.js

```diff
@@ -16,92 +16,133 @@
+        songAlbum: 'positions',
+        yearReleased: '2020',
+        youtubeLinkToSong: 'https://www.youtube.com/embed/apeXKjfP_yE'
+    },
+    {
+        songTitle: 'Pretty Savage',
+        artistName: 'Blackpink',
+        songAlbum: 'The Album',
+        yearReleased: '2020',
+        youtubeLinkToSong: 'https://www.youtube.com/embed/F8c8f2nK82w'
+    },
+    {
+        songTitle: 'Cardigan',
+        artistName: 'Taylor Swift',
+        songAlbum: 'Folklore',
+        yearReleased: '2020',
+        youtubeLinkToSong: 'https://www.youtube.com/embed/A4CVcQz0PuQ'
+    }];
+
+    // Pick a random song object
+    const song = getRandomElement(songs);
 
     // Add song to page
     const songContainer = document.getElementById('song-container');
 
-    songContainer.innerText = '\n"' + song.songTitle + '" - ' + song.artistName + '\n' 
-                                + song.songAlbum + ' (' + song.yearReleased + ')' + '\n\n'
-                                + 'To listen to the song on YouTube, click the photo of the song below!\n\n';
+    songContainer.innerText = '\n"' + song.songTitle + '" - ' + song.artistName + '\n'
+        + song.songAlbum + ' (' + song.yearReleased + ')' + '\n\n'
+        + 'To give the song a listen, click the video below!\n\n';
+
 
-    var chosenSong = document.createElement("img");
-    var songLink = document.createElement("a");
+    // Create an iframe element to embed the YouTube video into the portfolio page    
+    var songYT = document.createElement("iframe");
+
+    songYT.setAttribute("width", "560");
+    songYT.setAttribute("height", "315");
+    songYT.setAttribute("src", song.youtubeLinkToSong);
+    songYT.setAttribute("title", "YouTube video player");
+    songYT.setAttribute("frameborder", "0");
+    songYT.setAttribute("allow", "accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture");
+    songYT.setAttribute("allowfullscreen", "true");
+
+    songContainer.appendChild(songYT);
+}
 
-    chosenSong.setAttribute("src", song.albumCoverURL);
-    chosenSong.setAttribute("width", "250");
-    chosenSong.setAttribute("height", "250");
-    chosenSong.setAttribute("align", "center");
+function translateText() {
+    const message = document.getElementById('message').value;
+    const languageCode = document.getElementById('language').value;
 
-    console.log(song.albumCoverURL);
+    const translatedMessageContainer = document.getElementById('translated-message-container');
+    translatedMessageContainer.innerText = 'Translating...';
 
-    // Connect YouTube link to image and add into page
-    songLink.setAttribute("href", song.youtubeLinkToSong);
-    songLink.appendChild(chosenSong);
+    const params = new URLSearchParams();
+    params.append('message', message);
+    params.append('languageCode', languageCode);
+
+    fetch('/translate', {
+        method: 'POST',
+        body: params
+    }).then(response => response.text())
```

Comment:
Don't spend your time on it now, but in general prefer to use JSON for client-server requests. This is the standard for the web. 

A few reasons why:
 * Every language can convert to/from JSON; so defining an interface (e.g. your POST method on the TranslateServlet) in JSON makes it easier to communicate with any client - regardless of programming language.
 * JSON can represent simple or complex data structures. It's easy to add data (new fields) and can hold complex data structures, like arrays of objects that have many fields.


You'll find that just about every API accepts JSON.